src/app.js

```diff
@@ -1 +1,52 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+const countreis = document.getElementById('list of countries');
+const errorMessage = document.getElementById('error');
+
+async function fetchCountries(){
+    try {
+        const response = await axios.get('https://restcountries.com/v2/all');
+        const country = await response.data;
```

Comment:
Await is hier niet nodig.

Reply:
data is een inner object array, daarom kan je niet direct sorteren met array method. Dus er is Await nodig.

Reply:
Op regel 8 maak je een async request naar de api die data word opgeslagen in de const response, en die data is direct beschikbaar als je die aanspreekt gaat er niet opnieuw een request naar de countries api.